CONTRIBUTING.md

```diff
@@ -54,6 +54,21 @@ Create an issue on the repository and provide the following information by filli
 -   Notes (for example why you think this might be happening, or stuff you tried that didn't work)
+-   Issue severity
+
+### Issue severity classification
+
+Providing the correct issue severity classification allows us to more adequately assess and prioritize your issue.
+
+Please be aware of our issue severity classification criteria:
+
+| Severity level | Description                                                                                                                                                                                                                                                                                                                                           | Examples                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
+| -------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **SEV 1**      | A critical design or functionality issue that breaks the design system, causes significant usability problems, or exposes critical security vulnerabilities. This issue impacts all users and/or essential workflows, making users unwilling or unable to complete any tasks. Requires immediate attention.                                           | Broken navigation system, complete unresponsiveness on all devices, components not rendering, inaccessible primary actions, security vulnerabilities. Accessibility: The end user is not able to access the product or complete core user tasks or activities (key navigational elements not accessible via keyboard, missing or incorrect form labels that prevent screen reader users from completing forms or actions, critical color contrast issues that prevent users from reading or interacting with essential content). |
```

Comment:
```suggestion
| **SEV 1**      | A critical design or functionality issue that breaks the design system, causes significant usability problems, or exposes critical security vulnerabilities. This issue impacts all users and/or essential workflows, making users unable to complete any tasks. Requires immediate attention.                                           | Broken navigation system, complete unresponsiveness on all devices, components not rendering, inaccessible primary actions, security vulnerabilities. Accessibility: The end user is not able to complete core tasks or activities (key navigational elements not accessible via keyboard, missing or incorrect form labels that prevent screen reader users from completing forms or actions, critical color contrast issues that prevent users from reading or interacting with essential content). |
```
Minor wording adjustments.
Also, can we add a newline to the accessibility examples to make it easier to read?

Reply:
Updated.